packages/twenty-server/src/database/typeorm/typeorm.service.ts

```diff
@@ -37,27 +37,35 @@ export class TypeORMService implements OnModuleInit, OnModuleDestroy {
   public async connectToDataSource(
     dataSource: DataSourceEntity,
   ): Promise<DataSource | undefined> {
-    // Wait for a bit before trying again if another initialization is in progress
-    while (this.isDatasourceInitializing.get(dataSource.id)) {
-      await new Promise((resolve) => setTimeout(resolve, 10));
-    }
+    const isMultiDatasourceEnabled = false;
```

Comment:
this is to test with a single postgres datasource, it seems to provide a more stable performance but nothing crazy